setattributes.js

```diff
@@ -1,15 +1,8 @@
-module.exports = function setAttributes(element, attributes) {
-  var value;
-
-  for (var key in attributes) {
-    if (Object.prototype.hasOwnProperty.call(attributes, key)) {
-      value = attributes[key];
-
-      if (value == null) {
-        element.removeAttribute(key);
-      } else {
-        element.setAttribute(key, value);
-      }
+module.exports = function setAttributes(el, attrs) {
+  Object.entries(attrs).forEach(([name, value]) => {
```

Comment:
1. The rename from `key` to `name` better matches [the documentation on MDN][0]. Could you make a pull request to change that?
2. Is `Object.entries` slower than `for ... in`? I could imagine it being slower for large objects because it has to allocate an array and then destructure it for every iteration.

[0]: https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute

Reply:
1. Actually, the PR already use `name` and `value`. Isn’t it your first choice?

2. Good question. [This test](https://www.measurethat.net/Benchmarks/Show/7406/0/consume-objectentries-vs-forin-vs-forin-fn-22222#latest_results_block) is in favor of `Object.entries()`, but I’d say the difference isn’t significant. In the context of `setAttribute`, we’re likely not have big objects, but maybe I’m wrong on this point.